src/sentry/tasks/commit_context.py

```diff
@@ -155,76 +150,25 @@ def process_commit_context(
         with lock.acquire():
             metrics.incr("sentry.tasks.process_commit_context.start")
 
-            cache_key = DEBOUNCE_CACHE_KEY(group_id)
-
             set_current_event_project(project_id)
 
             project = Project.objects.get_from_cache(id=project_id)
             set_tag("organization.slug", project.organization.slug)
 
-            owners = GroupOwner.objects.filter(
-                group_id=group_id,
-                project=project,
-                organization_id=project.organization_id,
-                type=GroupOwnerType.SUSPECT_COMMIT.value,
-            )
             basic_logging_details = {
                 "event": event_id,
                 "group": group_id,
                 "organization": project.organization_id,
             }
-            # Delete old owners
-            to_be_deleted = owners.filter(
-                date_added__lte=django_timezone.now() - PREFERRED_GROUP_OWNER_AGE
-            )
-
-            if len(to_be_deleted):
-                for record in to_be_deleted:
-                    record.delete()
-
-            current_owners = owners.filter(
```

Comment:
No need to check for existing suspect commits anymore since this is only run once per issue